### PR TITLE
Simulate click by default

### PR DESCRIPTION
**Pull request by jperl**
Default click to use HTMLElement.click with is more stable than ElementHandle.click. Provide option to `{ simulate: false }`.

Relates to https://github.com/puppeteer/puppeteer/issues/3347

### CODE REVIEW

**Comment by jperl** [comment]
@flaurida Do you like { evaluate: false }, or { puppeteer: true } better?

**Comment by flaurida** [comment]
`{evaluate: false}` is my favorite as is most explicit and all use puppeteer

**Comment by jperl** [comment]
After updating the docs I think simulate is best.

File: docs/docs/api.md [hunk 1]
```diff
@@ -448,6 +449,7 @@ await browser.type(selectors[1], "my@email.com");
 [findelementoptions]: #interface-findelementoptions "FindElementOptions"
 [findpageoptions]: #interface-findpageoptions "FindPageOptions"
 [function]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function "Function"
+[htmlelement.click()]: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/click "HTMLElement.click"
```
Comment (flaurida) [inline docs/docs/api.md]: does capitalization matter here?

File: packages/browser/src/actions/index.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 export { clearElement } from "./clearElement";
-export { clickElement } from "./clickElement";
+export { clickElement, ClickOptions } from "./clickElement";
```
Comment (flaurida) [inline packages/browser/src/actions/index.ts]: abc

File: packages/browser/src/actions/clickElement.ts [hunk 3]
```diff
@@ -1,8 +1,21 @@
...
+  simulate?: boolean;
+};
+
+export const clickElement = async (
+  element: ElementHandle,
+  options: ClickOptions = {}
+): Promise<void> => {
+  logger.verbose(`clickElement: received ${JSON.stringify(options)}`);
+
+  if (options.simulate === false) {
```
Comment (flaurida) [inline packages/browser/src/actions/clickElement.ts]: i'd add a comment here and link to puppeteer issue to explain this

File: docs/docs/api.md (hunk 1 above)
Comment (jperl) [reply docs/docs/api.md]: Nope

**Pull request merged**